certora/specs/AccessControlDefaultAdminRules.spec

```
@@ -209,7 +209,8 @@ rule noPendingDefaultAdminChange(env e, method f, calldataarg args) {
   ) => (
     f.selector == beginDefaultAdminTransfer(address).selector ||
     f.selector == acceptDefaultAdminTransfer().selector ||
-    f.selector == cancelDefaultAdminTransfer().selector
+    f.selector == cancelDefaultAdminTransfer().selector ||
+    f.selector == renounceRole(bytes32,address).selector
```

As far as I can tell, `renounceRole(bytes32,address)` doesn't affect the pending admin, because it makes it 0 anyway. Was it failing?

```suggestion
f.selector == cancelDefaultAdminTransfer().selector
```

---

If we're keeping, we need to change the title of the rule and the error message for this assert to include renounce

---

Updated docs and message.

---

Answered in Slack, the reason we need this is because #4230, the `renounceRole`affects now the pending admin schedule, so the alternative is to have two rules (one using `renounceRole`, and the other not):

```
assert pendingAdminBefore != pendingAdminAfter => ( ... );
assert scheduleBefore != scheduleAfter => ( ... );
```

But we considered that unnecessary. Solving